src/utils/configs.js

```diff
@@ -22,9 +22,16 @@ let isAdmin = false;
   const el = document.querySelector(`meta[name='env:${x.toLowerCase()}']`);
   configs[x] = el ? el.getAttribute("content") : process.env[x];
 
-  if (x === "BASE_ASSETS_PATH" && configs[x]) {
+  if (x === "BASE_ASSETS_PATH" && configs["BASE_ASSETS_PATH"]) {
     // eslint-disable-next-line no-undef
-    __webpack_public_path__ = configs[x];
+    __webpack_public_path__ = configs["BASE_ASSETS_PATH"];
```

Comment:
Should this be below reassigning configs[BASE_ASSETS_PATH] to an absolute url on line 33?

Reply:
Yeah, seems to have the same effect, so might as well do it after for consistency.